sonar-db/src/main/java/org/sonar/db/version/v62/PopulateOrganizationUuidOfPermissionTemplates.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.db.version.v62;
+
+import java.sql.Date;
+import java.sql.SQLException;
+import org.sonar.api.utils.System2;
+import org.sonar.db.Database;
+import org.sonar.db.version.BaseDataChange;
+import org.sonar.db.version.MassUpdate;
+import org.sonar.db.version.Select;
+
+import static com.google.common.base.Preconditions.checkState;
+
+public class PopulateOrganizationUuidOfPermissionTemplates extends BaseDataChange {
+
+  private static final String INTERNAL_PROPERTY_DEFAULT_ORGANIZATION = "organization.default";
+
+  private final System2 system2;
+
+  public PopulateOrganizationUuidOfPermissionTemplates(Database db, System2 system2) {
+    super(db);
+    this.system2 = system2;
+  }
+
+  @Override
+  public void execute(Context context) throws SQLException {
+    String organizationUuid = selectDefaultOrganizationUuid(context);
+
+    MassUpdate massUpdate = context.prepareMassUpdate();
```

Comment:
since there can't be that many templates, I think it would be more efficient while not risky to use a single update `update set  organization_uuid=?, updated_at=? where organization_uuid is null`